site/content/docs/2.0/extend/icons.md

```diff
@@ -17,9 +17,170 @@ Quickstart, and Arizona Bootstrap.
 - [Arizona Icons](https://github.com/az-digital/az-icons) (Arizona branded icons)
 - [Google Material icons (Sharp)](https://material.io/resources/icons/?style=sharp)
 
+## Arizona Icons Implementation
+
+Include a reference to the Arizona Icons' CDN to your project in order to use the font icons: https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css
```

Comment:
Might be good to wrap this link in the same syntax as the other CND references (to make it easier to copy/paste the link).
```suggestion
Include a reference to the Arizona Icons' CDN to your project in order to use the font icons: 
```html
https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css
```
```

Reply:
GitHub isn't picking up on the syntax correctly... But there should be a closing ``` on line 26.

Reply:
You mean here? https://raw.githubusercontent.com/az-digital/arizona-bootstrap/main/site/content/docs/2.0/extend/icons.md

Reply:
Updated!